pubspec.yaml

```diff
@@ -119,6 +119,9 @@ dependencies:
   # hive
   hive: 2.0.4
 
+  # Detect whether widget is rendered
+  visibility_detector: 0.2.2
```

Comment:
IMO it should be belonged to `dev_dependencies`?

Reply:
it should because we need a lib that can detect whether the widget is built, I suggest we customize our own.

Reply:
I mean, "how about move it to `dev_dependencies`?"

Reply:
I think because dev_dependencies are dependencies that are not available for code in the resulting application, but only for tests, examples, tools, or to add executable tools like for code generation to your project. Since this library is used in the final app => I don't think it should at dev_dependencies.